### ISSUE TITLE
Animation mods on Character Collections Not applying

### ISSUE THREAD

**Issue by TheodoreKaiju** [xivdev/Penumbra#211]
With Character Collections, mod that change animations dont work as of this patch. They worked fine as of last patch. They apply in the default collection but do not apply in the character collections

**Comment by TheodoreKaiju** [comment]
For context the animation im using is just an idle/walking animation swap from one race to another

**Comment by munkviken** [comment]
They work as intended right now in the penumbra rework. You need them enabled in both your default and character collection.

Making them work in character collections is a current work in progress.

**Comment by TheodoreKaiju** [comment]
Wouldn’t doing this give every one else the said animation? Last update it
didn’t give the animation to everyone else but now it does.

**Comment by munkviken** [comment]
That is exactly what would happen.

**Comment by TheodoreKaiju** [comment]
Yeah I don’t want it to apply to EVERY other hrothgar. Last update it
didn’t do this and the animations worked just fine in character collections…

**Comment by munkviken** [comment]
I've already told you that feature is currently not available and is a work in progress.

Penumbra was completely reworked in how it works in the backend as of 0.5 and things like character exclusive animation mods are not fully supported yet.

**Comment by SerenitySunfire** [comment]
Also, if you seemed to have had animations working on a character collection, it was an accident and happened due to hacky reasoning. The current release removed the hacky elements of Penumbra, and for now and maybe moving forward, you need to have them on both Default and your collection.

**Comment by TheodoreKaiju** [comment]
Thank you for the clarification.

I suppose those hacky reasons were SOMEHOW helping me and my friends use
animation mods and having them only be applied to specific characters.

Ashame that I won’t be able to use them without applying them to everyone;
I wish y’all luck on y’all’s endeavors in working that feature in though

**Comment by Ottermandias** [comment]
I'm relatively sure I figured out a way to do it, I'll just have to test it for a while because breaking animations is dangerous.

**Comment by TheodoreKaiju** [comment]
:OO I’ll keep my eyes open
